api/services/lau.ts

```diff
@@ -126,11 +126,10 @@ async function createAccessLogFromRequest(req: EnhancedRequest) {
       requestType: (process === 'specific-access') ? REQUEST_TYPE.SPECIFIC : REQUEST_TYPE.CHALLENGED,
       caseRef: requestedRole?.attributes?.caseId,
       userId: requestedRole?.actorId,
-      action: process === 'specific-access' ? ACTION_TYPE.CREATED : ACTION_TYPE.APPROVED,
+      action: process === 'specific-access' ? ACTION_TYPE.CREATED : ACTION_TYPE.AUTO_APPROVED,
       timestamp: requestedRole?.notes[0]?.time,
       reason: reason,
-      requestStartTimestamp: requestedRole?.notes[0]?.time,
-      requestEndTimestamp: requestedRole?.endTime
+      requestEndTimestamp: null
```

Comment:
Why is the end timestamp null? It should be the end of the granted access

Reply:
There is no way user can specify for how long he is requesting access. By default it is set to 1 month from now, which is misleading on LAU, because that is not what user requested